BacktraceScreen.h

```diff
@@ -0,0 +1,47 @@
+#ifndef HEADER_BacktraceScreen
+#define HEADER_BacktraceScreen
+/*
+htop - BacktraceScreen.h
+(C) 2023 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h" // IWYU pragma: keep
+
+#if defined(HAVE_BACKTRACE)
```

Comment:
Not generating any interface when the backtracing is disabled doesn't look right to me. Have those checks on the `BacktraceScreen.c` side of things.